doc/developer/design/20220330_persist.md

```diff
@@ -381,13 +445,10 @@ extended as the reader or writer is used. If the lease expires (or the process
 is restarted), the reader/writer acquires a new lease under a new (internal)
 identifier.
 
-WIP Feels like there's more to say here, but I'm not exactly sure what. Maybe
```

Comment:
want to bump that the protocol we've described is lock-free. As long as the consensus and blob services are up, at least one writer/reader will always make progress. (we're in the single-compare-and-swap universal class of algorithms which is not a commonly used term but its defined on pg 3 here https://arxiv.org/pdf/1311.3200.pdf )